share/man/man9/pmap_kextract.9

```diff
@@ -0,0 +1,56 @@
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.\" Copyright (c) 2023 The FreeBSD Foundation
+.\"
+.\" This manual page was written by Mina Galić <FreeBSD@igalic.co> under
+.\" sponsorship from the FreeBSD Foundation.
+.\"
+.Dd August 22, 2023
+.Dt PMAP_KEXTRACT 9
+.Os
+.Sh NAME
+.Nm pmap_kextract ,
+.Nm vtophys
+.Nd extract from kernel page table to a physical address
+.Sh SYNOPSIS
+.In sys/param.h
+.In vm/vm.h
+.In vm/pmap.h
+.Ft vm_paddr_t
+.Fo pmap_kextract
+.Fa "vm_offset_t va"
+.Fc
+.Ft vm_paddr_t
+.Fo vtophys
+.Fa "vm_offset_t va"
+.Fc
+.Sh DESCRIPTION
+The
+.Fn pmap_kextract
+function and its alias
+.Fn vtophys
+map a kernel virtual address to a physical page.
+.Pp
+.Sh RETURN VALUES
+The
+.Fn pmap_kextract
+function will return the physical page address associated with the
+kernel virtual address
+.Fa va
+inside the physical map of the address space.
+In case of failure,
+.Fn pmap_kextract
+will return
+.Dv NULL .
+.Sh SEE ALSO
+.Xr mutex 9 ,
```

Comment:
Why is this needed?

Reply:
that looks like a left-over from copying pmap_extract.9